client/src/components/internship/EditInternship.vue

```diff
@@ -7,58 +8,56 @@
           <div class="col">
             <label for="startDate">
               {{ $t("internship.form.startDate") }}
-              <span v-if="startDate">({{ $t("internship.form.currently") }}: {{startDate}})</span>
             </label>
-            <input v-model="newStartDate"
+            <input v-model="startDate"
                    type="date"
                    id="startDate"
                    class="form-control"
-                   :placeholder="startDate"/>
+                  />
           </div>
           <div class="col">
-            <label for="startDate">
+            <label for="endDate">
               {{ $t("internship.form.endDate") }}
-              <span v-if="endDate">({{ $t("internship.form.currently") }}: {{endDate}})</span>
             </label>
-            <input v-model="newEndDate"
+            <input v-model="endDate"
                    type="date"
                    id="endDate"
                    class="form-control"
-                   :placeholder="endDate"/>
+                   />
           </div>
         </div>
 
         <div class="row my-4">
           <div class="col">
             <label for="operationalArea">{{ $t('internship.form.operationalArea') }}</label>
-            <input v-model="newOperationalArea"
+            <input v-model="internship.operationalArea"
                    type="text"
                    class="form-control"
                    id="operationalArea"
-                   :placeholder="operationalArea ?? $t('internship.form.operationalArea')"/>
+                   :placeholder="internship.operationalArea ?? $t('internship.form.operationalArea')"/>
```

Comment:
Dadurch, dass die ganzen Inputs jetzt die eigentliche Internship Property modeln, wird die auch immer direkt geupdated, wenn man den Input ändert. Placeholder werden aber nur bei leerem Input angezeigt. Wenn jetzt die Property durch clearen des Inputs geleert wird, sollte ein Placeholder angezeigt werden, aber der wird auch gecleart, da er ja mit gemodelt wird 😅 (Gilt für alle Placeholder hier in EditInternship) Von daher könnten die dann auch einfach weg, würde ich sagen.